tests/fixtures/functionLayersSourceChange/serverless.yml

```diff
@@ -0,0 +1,16 @@
+service: service
```

Comment:
It'll be nice to introduce just one fixture

Reply:
I wasn't sure exactly how to do this with one fixture. The test is to check that if only a line of the layer code is changed then a deploy will create a new lambda version that points to the new layer created. Everything else (serverless.yml, directory/file names etc.) all stays the same.

One way I considered was extending the fixture to point to a second layer directory that was slightly different. However, in that case the layer properties would be different and so it wouldn't recreate the exact scenario being tested for, i.e. the serverless.yml would be different.

I looked to see if there was a way I could update the code in the fixture's layer at runtime but that doesn't seem right. Any ideas?

Reply:
So if I understand correctly, in test, we should modify same layer file and proceed with second deployment. It's important to not change the filename of a layer as that wouldn't properly reflect what we want to test.

If my understanding is correct, then maybe we could create two different layer versions (e.g. `testLayer1`, and `testLayer2`) directly in a fixture, and before first deployment copy `testLayer1` into `testLayer` and before second deployment  remove `testLayer` and copy `testLayer2` into `testLayer`)

Cleanup of that can be done through `fixtures.cleanup({ extraPaths: ['testLayer'])`

Reply:
Thanks, this should be done now.